src/Files.App/UserControls/StatusCenter/StatusCenter.xaml

```diff
@@ -227,7 +227,7 @@
 							<!--  Graph  -->
 							<Grid
 								x:Name="MainGraphCartesianChartClipGrid"
-								Height="72"
+								Height="128"
```

Comment:
Is this increased height by design?

Reply:
I put it there since it was easier to see in testing and I thought it looked nicer but it's not necessary

Reply:
I know this was discussed on Discord, what was the conclusion?

Reply:
72 is what the design uses, but the height should balance out the heading padding for the speed text and the chart height.

We are not trying to match File Explorer, and we should be conscious of how tall the flyout becomes with multiple operations on-going and the expanded size.